src/render/components/tag/tagStructure.js

```diff
@@ -274,10 +283,10 @@ const renderTagStructure = (tag, metaIndex) => {
         The following information is unique to the <strong>${tag.name}</strong> tag.
       </p>
     `)}
-    ${structView(tag.invaderStruct, tag.invaderStructName, tag.comments, metaIndex, addHeading, 2, true)}
+    ${structView(tag.invaderStruct, tag.invaderStructName, tag.comments, metaIndex, addHeading, addSearchText, 2, true)}
```

Comment:
The strategy here is that while we recursively render the tag structure to HTML, we pass down some callback functions `addHeading` and `addSearchText` which the rendering code can use to add heading and searchable information at the same time.